crates/ruff/tests/snapshots/show_settings__display_default_settings.snap

```diff
@@ -284,23 +301,23 @@ linter.flake8_type_checking.runtime_required_base_classes = []
 linter.flake8_type_checking.runtime_required_decorators = []
 linter.flake8_type_checking.quote_annotations = false
 linter.flake8_unused_arguments.ignore_variadic_names = false
-linter.isort.required_imports = []
+linter.isort.required_imports = {}
 linter.isort.combine_as_imports = false
 linter.isort.force_single_line = false
 linter.isort.force_sort_within_sections = false
 linter.isort.detect_same_package = true
 linter.isort.case_sensitive = false
 linter.isort.force_wrap_aliases = false
-linter.isort.force_to_top = []
+linter.isort.force_to_top = {}
 linter.isort.known_modules = {}
 linter.isort.order_by_type = true
 linter.isort.relative_imports_order = furthest_to_closest
-linter.isort.single_line_exclusions = []
+linter.isort.single_line_exclusions = {}
 linter.isort.split_on_trailing_comma = true
-linter.isort.classes = []
-linter.isort.constants = []
-linter.isort.variables = []
-linter.isort.no_lines_before = []
+linter.isort.classes = {}
+linter.isort.constants = {}
+linter.isort.variables = {}
+linter.isort.no_lines_before = {}
```

Comment:
These are sets. We could represent them as `[]` since that's how they're presented in TOML. I don't feel strongly.

Reply:
I'd prefer to distinguish them from an empty key/value structure so `[]` seems better.

Reply:
Sounds good, although now they're not distinguished from an empty array :)